tests/test.display.js

```diff
@@ -407,7 +402,7 @@ describe('Display/Canvas Helper', function () {
         });
 
         it('should wait until an image is loaded to attempt to draw it and the rest of the queue', function () {
-            var img = { complete: false };
+            var img = { complete: false, addEventListener: sinon.spy() }
```

Comment:
That would be nice, but I'm not sure how to robustly make all browsers delay loading.